modules/using-ccoctl-create-iam-role-alb-operator.adoc

```diff
@@ -6,7 +6,7 @@
 [id="using-ccoctl-create-iam-role-alb-operator_{context}"]
 = Creating an AWS IAM role by using the Cloud Credential Operator utility
 
-You can use the Cloud Credential Operator utility (`ccoctl`) to create an AWS IAM role for the AWS Load Balancer Operator. An AWS IAM role is used to interact with subnets and Virtual Private Clouds (VPCs).
+You can use the Cloud Credential Operator utility (`ccoctl`) to create an {aws-short} IAM role for the {aws-short} Load Balancer Operator. An {aws-short} IAM role is used to interact with subnets and Virtual Private Clouds (VPCs).
```

Comment:
Not sure is you can remove the passive voice here. Maybe: " An {aws-short} IAM role interacts with subnets and Virtual Private Clouds (VPCs).

Reply:
:heavy_check_mark: